filebeat/module/kafka/_meta/docs.asciidoc

```diff
@@ -1,24 +1,72 @@
+:modulename: kafka
+
 == Kafka module
 
-This module collects and parses the logs created by https://kafka.apache.org/[Kafka].
+The +{modulename}+ module collects and parses the logs created by
+https://kafka.apache.org/[Kafka].
+
+include::../include/what-happens.asciidoc[]
 
 [float]
 === Compatibility
 
-The Kafka module was tested with logs from versions 2.11.
+The +{modulename}+ module was tested with logs from versions 2.11.
+
+include::../include/running-modules.asciidoc[]
 
 [float]
-=== Dashboard
+=== Example dashboard
 
-This module comes with a sample dashboard to see Kafka logs and stacktraces.
+This module comes with a sample dashboard to see Kafka logs and stack traces.
 
+[role="screenshot"]
 image::./images/filebeat-kafka-logs-overview.png[]
 
-[float]
-=== Logs fileset settings
+include::../include/configuring-intro.asciidoc[]
+
+The following example shows how to set paths in the +modules.d/{modulename}.yml+
+file to override the default paths for logs:
+
+["source","yaml",subs="attributes"]
+-----
+- module: kafka
+  log:
+    enabled: true
+    var.paths:
+      - "/path/to/logs/controller.log*"
+      - "/path/to/logs/server.log*"
+      - "/path/to/logs/state-change.log*"
+      - "/path/to/logs/kafka-*.log*"
+-----
+
+
+// REVIEWERS: I must be doing something wrong with the config settings. The
```

Comment:
See above.

Reply:
Yah, I was in a hurry to get this work done and forgot to remove the notes. I'll create an issue to cleanup the notes.